src/mds/Server.cc

```diff
@@ -10271,6 +10272,7 @@ void Server::handle_client_mksnap(MDRequestRef& mdr)
   }
   if (diri->is_system() && !diri->is_root()) {
     // no snaps in system dirs (root is ok)
+    dout(5) << "is a system dir" << dendl;
```

Comment:
could we have: `is an internal system dir` instead ?

Reply:
sure, done